spring-web/src/main/java/org/springframework/web/bind/annotation/ExceptionHandler.java

```diff
@@ -129,4 +130,9 @@
 	 */
 	Class<? extends Throwable>[] value() default {};
 
+	/**
+	 * Exceptions excluded from handling by the annotated method.
+	 * @since 5.0
```

Comment:
I don't know. 
The Jira issue has Fix Version as '5.0 Backlog' so I thought it's planned for 5.0.